tests/phpunit/includes/testcase-trait.php

```diff
@@ -65,6 +65,13 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) { // p
 		remove_action( 'current_screen', '_load_remote_block_patterns' );
 		remove_action( 'current_screen', '_load_remote_featured_patterns' );
 
+		if ( version_compare( $GLOBALS['wp_version'], '6.4', '>=' ) ) {
```

Comment:
We don't really need this check. Do we ? As far as I know we never tests emojis :wink: :roll_eyes:

Reply:
I thought it could be used by a third party running along our test suite for WP < 6.4. But maybe a bit overkill.
Removed in 95cc1901.